functionary/ui/urls.py

```diff
@@ -140,6 +140,11 @@
         (scheduling.ScheduledTaskCreateView.as_view()),
         name="new-schedule",
     ),
+    path(
```

Comment:
I'm assuming this got accidentally removed in some other PR or merge conflict process?

Reply:
Yea, I may have accidentally removed it when rebasing the team/environment management stuff. Although, these urls are going to be refactored in #167